actions/matrixsynapse

```diff
@@ -80,6 +87,24 @@ def subcommand_disable(_):
     action_utils.service_disable('matrix-synapse')
 
 
+def subcommand_enable_user_registrations(_):
+    """Enable new user registrations on the matrixsynapse """
+    subprocess.call([
+        "sed", "-i",
+        "s/enable_registrations: false/enable_registrations: true/g",
+        MATRIXSYNAPSE_YAML
+    ])
```

Comment:
As a minor point, using `sed` isn't Pythonic - better to use Python's YAML abstractions. That also makes less assumptions about the option being present in the first place. See current code, if you like.